modules/local/samtools/flagstat/main.nf

```diff
@@ -0,0 +1,23 @@
+process SAMTOOLS_FLAGSTAT {
```

Comment:
I'm a little confused why there are so many modules on nf-core that are local instead.

Reply:
See above; 
> Yes, I will see why I am using my local version (which is an older nf-core version) again. I remember trying to use the nf-core module and something did not work anymore, but I will take another look,